workloads/network-perf/common.sh

```diff
@@ -3,7 +3,12 @@ set -x
 
 _es=search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com
 _es_port=80
+_es_baseline=search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com
+_es_baseline_port=80
+throughput_tolerance=5
+latency_tolerance=5
 _metadata_collection=true
+client_server_pairs=(1 2 4)
```

Comment:
Can we default to 1? And in our CI if we need something different we just update this?

Reply:
Ah, I see later there is a dependence on this..

Reply:
the baremetal CI uses 1,2 and 4 pairs as default I guess
we can change this manually as our requirement